pom.xml

```diff
@@ -467,7 +467,7 @@
         <maven.buildnumber.plugin.version>1.4</maven.buildnumber.plugin.version>
         <org.apache.felix.annotations.version>1.2.4</org.apache.felix.annotations.version>
         <identity.governance.version>1.3.15</identity.governance.version>
-        <carbon.identity.framework.version>5.16.95</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.16.108-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
Use the latest released version. Do not use the SNAPSHOT version

Reply:
Fixed